block/style.scss

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
Also not sure it's worth including this if it's not doing anything.  We can keep the plugin simpler with less junk clutter files by not including them if they're not used.